lib/v2/swufe/router.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (router) {
+    router.get('/jwc/:category', require('./jwc'));
+    router.get('/yz/:category', require('./yz'));
+    router.get('/www/:category', require('./www'));
```

Comment:
https://github.com/DIYgod/RSSHub/blob/92d9c57cbe02e9deb6298642fbeba1616573a01a/lib/v2/swufe/jwc.js#L11
https://github.com/DIYgod/RSSHub/blob/92d9c57cbe02e9deb6298642fbeba1616573a01a/lib/v2/swufe/www.js#L12
```suggestion
    router.get('/jwc/:category?', require('./jwc'));
    router.get('/yz/:category', require('./yz'));
    router.get('/www/:category?', require('./www'));
```